main.py

```diff
@@ -1,10 +1,58 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def load_data_from_json(filename: str) -> dict:
+    with open(filename) as file:
+        data = json.load(file)
+        return data
+
+
+def process_players_data(data: dict) -> None:
+    for nickname, player_data in data.items():
+        race_data = player_data["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data.get("description", "")}
+        )
+
+        guild_data = player_data.get("guild")
+        if guild_data:
+            guild_name = guild_data.get("name", "")
+            guild_description = guild_data.get("description", None)
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_name,
+                defaults={"description": guild_description}
+            )
+        else:
+            guild = None
```

Comment:
You can create this viriable at the start and do not use else block .

If your condition is triggered, the contents of the variable will simply be overwritten